ibis/tests/all/test_temporal.py

```diff
@@ -78,12 +76,12 @@ def test_date_truncate(backend, alltypes, df, unit):
     backend.assert_series_equal(result, expected)
 
 
-@pytest.mark.parametrize(
-    'unit',
-    ['Y', pytest.mark.xfail('Q'), 'M', 'W', 'D', 'h', 'm', 's',
-     pytest.mark.xfail('ms'), pytest.mark.xfail('us')
-     ])
+@pytest.mark.parametrize('unit', [
+    'Y', pytest.mark.xfail('Q'), 'M', 'W', 'D',
+    'h', 'm', 's', pytest.mark.xfail('ms'), pytest.mark.xfail('us')
+])
 @tu.skipif_unsupported
+@pytest.mark.filterwarnings('ignore')
```

Comment:
Can we explicitly ignore warnings inside the function instead of ignoring every possible one that arises from any of the test code?

Also, can you add a comment about which warning we're trying to ignore?